backends/magma/ceed-magma.c

```diff
@@ -726,6 +884,33 @@ static int CeedQFunctionDestroy_Magma(CeedQFunction qf) {
 }
 
 static int CeedQFunctionCreate_Magma(CeedQFunction qf) {
+  if (strstr(qf->focca, "t30-operator.c:setup") !=NULL)
```

Comment:
This QFunction code is simply unacceptable. We cannot allow a backend to only support qfunctions that are in the test suite. Users *must* be able to write and use their own QFunctions.

@jedbrown, I am leaving it as is in my rebase pass, but I'm not sure how we're going to address this.

Reply:
Agreed. We've discussed it a few times. The plan has been for this backend to use QFunctions processed by the CUDA backend, which did not exist at the time this PR was submitted, but slow activity in this branch has left it always playing catch-up. We should probably merge this as soon as it doesn't cause more breakage than before, but implementing the CUDA QFunctions should be considered urgent by the MAGMA backend developers.

Reply:
Fair enough. I wanted to make sure we were aware because this is a regression in this backend's,ability to handle qfunctions.